File: sample-anim-gif-prep/sample.js
const BaseClass = require( './BaseClass' );
const { aFunc, bFunc } = require( './AModule' );

class AClass extends BaseClass.ABaseClass {

    constructor( val1, val2 ) {
        super( val1 );
        this.val2 = val2;
    }

    meth1( aval ) {
        let val = this.val1 * this.val2 + aval;
        return super.meth1( val );
    }

    meth2( astr ) {
        return func( astr, this.val1, this.val2 );
    }
}

function func( a, b, c ) {
    let x = aFunc( a, b );
    return bFunc( c, x );
}

File: sample-anim-gif-prep/sample.py
import BaseClass
from AModule import aFunc, bFunc

class AClass( BaseClass.ABaseClass ):

    def __init__( self, val1, val2 ):
        super( ABaseClass, self ).__init__( val1 )
        self.val2 = val2

    def meth1( self, aval ):
        val = self.val1 * self.val2 + aval
        return super( ABaseClass, self ).meth1( val )

    def meth2( self, astr ):
        return func( astr, self.val1, self.val2 );


def func( a, b, c ):
    x = aFunc( a, b )
    return bFunc( c, x )









